src/Web/Microsoft.NET.Sdk.Web.Targets/Sdk.DefaultItems.targets

```diff
@@ -0,0 +1,143 @@
+<!--
+***********************************************************************************************
+Sdk.DefaultItems.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<!--
+    Determine the version (including patch) of ASPNET Core to target.
+
+    When targeting ASPNET Core, the TargetFramework is used to specify the major and minor version of the runtime to use.
+    By default, the patch version is inferred. The general logic is that self-contained apps will target the latest patch
+    that the SDK knows about, while framework-dependent apps will target the ".0" patch (and roll forward to the latest
+    patch installed at runtime).
+
+    The TargetLatestAspNetCoreRuntimePatch property can be set to true or false to explicitly opt in or out of the logic
+    to roll forward to the latest patch, regardless of whether the app is self-contained or framework-dependent. The
+    default value of this property is TargetLatestRuntimePatch set by the dotnet SDK.
+
+    The AspNetCoreAppRuntimeFrameworkVersion and AspNetCoreAllRuntimeFrameworkVersion are where the actual versions of the
+    ASPNET Core runtimes to target is stored. They are the versions that are used in the implicit PackageReference to
+    Microsoft.AspNetCore.App and Microsoft.AspNetCore.All. The AspNetCoreAppRuntimeFrameworkVersion and
+    AspNetCoreAllRuntimeFrameworkVersion can also be set explicitly, which will disable all the other logic that
+    automatically selects the version of .NET Core to target.
+
+    The framework version that is written to the runtimeconfig.json file is based on the actual resolved package version
+    of Microsoft.AspNetCore.App or Microsoft.AspNetCore.All. This is to allow floating the verion number (i.e. the user
+    could set the version to "2.0-*".
+
+  -->
+  <Choose>
+    <When Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'" >
+
+      <!-- Default patch version of .All Framework -->
+      <PropertyGroup Condition="'$(DefaultAspNetCoreAllPatchVersion)' == ''">
```

Comment:
If you are going to support the .All package as well as the .App package, then I'd suggest also supporting it when targeting .NET Core / ASP.NET Core 2.0.  Then you'll be able to to test the roll-forward logic when the target framework doesn't match the bundled version, making it more likely that everything will work when we start working on the next minor version of ASP.NET Core.

Reply:
The mechanism for the .All package is different for 2.0 since it uses the runtime store and not the shared framework. It would be a major design change and isn't something we want to do right now.